seatunnel-connectors-v2/connector-sentry/src/main/java/org/apache/seatunnel/connectors/seatunnel/sentry/sink/SentrySinkFactory.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.sentry.sink;
+
+import org.apache.seatunnel.api.configuration.util.OptionRule;
+import org.apache.seatunnel.api.table.factory.TableSinkFactory;
+
+public class SentrySinkFactory implements TableSinkFactory {
```

Comment:
```suggestion
@AutoService(Factory.class)
public class SentrySinkFactory implements TableSinkFactory {
```

Reply:
Hi, @EricJoy2048 It looks like this issue has not been resolved

Reply:
> Hi, @EricJoy2048 It looks like this issue has not been resolved

After discuss, we need use `@AutoService(TableSinkFactory.class)` replace `@AutoService(Factory.class)`

Reply:
@CalvinKirs  replace to `@AutoService(Factory.class)` done.